modules/oceanbase-ce/src/main/java/org/testcontainers/containers/OceanBaseContainer.java

```diff
@@ -0,0 +1,101 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+
+/**
+ * Testcontainers implementation for OceanBase.
+ * <p>
+ * Supported image: {@code oceanbase/oceanbase-ce}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>SQL: 2881</li>
+ *     <li>RPC: 2882</li>
+ * </ul>
+ */
+public class OceanBaseContainer extends JdbcDatabaseContainer<OceanBaseContainer> {
+
+    static final String NAME = "oceanbase";
+
+    static final String DOCKER_IMAGE_NAME = "oceanbase/oceanbase-ce";
+
+    static final String DEFAULT_TAG = "4.1.0.0";
```

Comment:
remove this

Reply:
You mean moving this default tag to the container provider?